### ISSUE TITLE
Narrator does not announce associated label with “Decoration” combobox and “State” combobox.

### ISSUE THREAD

**Issue by azaslonov** [paperbits/paperbits-demo#112]
User Experience:

If screen reader is not reading associated label with controls than screen reader user will not be able to find out what kind of information is needed in the form fields and will face difficulties while navigating on the page.

Note: User credentials should NOT be included in the bug.

 

Repro-Steps

1. Open Azure portal with valid credentials and create a API Management.

2. Click Developer Portal button to navigate to the portal.

3. Wait portal to load in design mode

4. Navigate to the Left nav and select "Styles" button.

5. Navigate to style page and select "Add variation" button.

6. Enable screen reader and navigate to the various controls of the "Add variation" dialog box.

7. Verify the issue.

 

Actual Result:

Narrator does not announce associated label with “Decoration” combo box and “State” combo box.

 

Expected result:

Narrator should announce the associated label with “Decoration” combo box and “State” combo box.

Note:

This issue does not exist with Chrome + JAWS and Chrome + NVDA 

Similar issue exist throughout the application for same controls.

 

MAS Reference:

MAS 4.2.1 - Object Information (33A)

 

Reference Links

Accessibility Insights! - Identify accessibility bugs before check-in and make bug fixing faster and easier.
External Bug Process: If this bug belongs to external team, mark it as resolved/done and assign it back to the tester with notes on where to file/route the bug. For more information please use this Link. “HCL Staff should not log any third party external bugs. Should be routed to edad team”.
Please reach out to C&AI Teams channel for any process related queries.

**Comment by azaslonov** [comment]
Fixed.
